katharsis-client/src/test/java/io/katharsis/client/AbstractClientTest.java

```diff
@@ -0,0 +1,64 @@
+package io.katharsis.client;
+
+import java.util.concurrent.TimeUnit;
+
+import javax.ws.rs.ApplicationPath;
+import javax.ws.rs.core.Application;
+
+import org.glassfish.jersey.server.ResourceConfig;
+import org.glassfish.jersey.test.JerseyTest;
+import org.junit.Before;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import io.katharsis.client.mock.models.Project;
+import io.katharsis.client.mock.models.Task;
+import io.katharsis.client.mock.repository.TaskRepository;
+import io.katharsis.locator.SampleJsonServiceLocator;
+import io.katharsis.queryParams.DefaultQueryParamsParser;
+import io.katharsis.queryParams.QueryParamsBuilder;
+import io.katharsis.rs.KatharsisFeature;
+import io.katharsis.rs.KatharsisProperties;
+
+public class AbstractClientTest extends JerseyTest {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add some tests to this class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2187)